File: templates/hexagonal-spring-boot/bootstrap/src/main/java/packageName/boot/ExampleApplication.java
package packageName.boot;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@ComponentScan(basePackages = "packageName")
public class ExampleApplication {

  public static void main(String[] args) {
    SpringApplication.run(ExampleApplication.class, args);
  }
}

File: templates/hexagonal-spring-boot-kotlin/bootstrap/src/main/kotlin/packageName/boot/ExampleApplication.kt
package packageName.boot

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan

@SpringBootApplication
@ComponentScan(basePackages = ["packageName"])
class ExampleApplication {
}

fun main(args: Array<String>) {
    SpringApplication.run(ExampleApplication::class.java, *args)
}
